src/KOKKOS/pppm_kokkos.h

```diff
@@ -36,30 +36,30 @@ KSpaceStyle(pppm/kk/host,PPPMKokkos<LMPHostType>);
 // fix up FFT defines for KOKKOS with CUDA and HIP
 
 #ifdef KOKKOS_ENABLE_CUDA
```

Comment:
@akohlmey I think this code should be deleted now, do you agree?

Reply:
No. Only the CMake build is set up to have only one FFT choice for either KOKKOS and non-KOKKOS. For the GNU make build we have to assume any mix of FFT "overlapping" FFT defines.

With the changes in this PR, it may be better to move those to `lmpfftsettings.h` which should be the canonical file to guarantee consistent FFT settings and defines.

Reply:
This is a bit like:
```
// if a user sets FFTW, it means FFTW3

#ifdef FFT_FFTW
#ifndef FFT_FFTW3
#define FFT_FFTW3
#endif
#endif
```

To this block which we should probably now also add an `#undef FFT_FFTW` for increased consistency.
I have been pondering to add statements to error out for incompatible choices.
Ultimately, I've chosen to wait a bit longer until FFT support for Intel GPUs in KOKKOS is sorted out and possibly HeFFTe support extended to support Kokkos and then integrated into LAMMPS.
After that would be a good point to look at all possible combination of options and map out what should be allowed, preferred, or disallowed.

Reply:
OK thanks for the explanation